share/commands/agent-reset

```diff
@@ -42,6 +42,12 @@ done
 rm -rf /var/rudder/cfengine-community/inputs/*
 cp -r /opt/rudder/share/initial-promises/* /var/rudder/cfengine-community/inputs
 
+# - restore initial ncf
+rm -rf /var/rudder/ncf/common/*
+rm -rf /var/rudder/ncf/local/*
+cp -r /opt/rudder/share/initial-ncf/* /var/rudder/ncf/common/
```

Comment:
Standard, yes. But standard for use with CFEngine (port 5308, _log_default), not with Rudder. See Normation/rudder-packages#1001.